Doc: Overview of nrf5340_audio features (#8803)
nrfconnect/sdk-nrf: doc/nrf/software_maturity.rst

.. _software_maturity_application_nrf5340audio_table:

.. toggle::

   .. list-table:: nRF5340 Audio application feature support
      :header-rows: 1
      :align: center
      :widths: auto

      * - Feature
        - Description
        - Limitations
        - Maturity level
      * - **Broadcast source**
        - Transmitting broadcast audio using Broadcast Isochronous Stream (BIS) and Broadcast Isochronous Group (BIG).

          Play and pause emulated by disabling and enabling stream, respectively.
        - The following limitations apply:

          * Basic Audio Profile (BAP) broadcast, one BIG with two BIS streams.
          * Audio input: USB or I2S (Line in or using Pulse Density Modulation).
          * Configuration: 48 kHz, 16 bit, several bit rates ranging from 24 kbps to 160 kbps.

        - Experimental
      * - **Broadcast sink**
        - Receiving broadcast audio using BIS and BIG.

          Synchronizes and unsynchronizes with the stream.
        - The following limitations apply:

          * BAP broadcast, one BIG, one of the two BIS streams (selectable).

>           * Audio output: I2S (Headset output).

Author: Changed

          * Configuration: 48 kHz, 16 bit, several bit rates ranging from 24 kbps to 160 kbps.

        - Experimental
      * - **Unicast source**
        - BAP unicast, one Connected Isochronous Group (CIG) with two Connected Isochronous Streams (CIS).

          Transmitting unidirectional audio using CIG and CIS.

          Play and pause emulated by disabling and enabling stream, respectively.
        - The following limitations apply:

          * BAP unicast, one CIG with two CIS.
          * Audio input: USB or I2S (Line in or using Pulse Density Modulation).
          * Configuration: 48 kHz, 16 bit, several bit rates ranging from 24 kbps to 160 kbps.

        - Experimental
      * - **Unicast sink**
        - BAP unicast, 1 CIG with 2 CIS streams.

>           Context type setting (for forcing enabling and disabling streaming, to emulate play and pause).

Reviewer: ```suggestion
          Play and pause emulated by forcing enabling and disabling streaming by changing available contexts.
```

Reviewer 2: With your edit, it is unclear what is emulated by what due to double "by" structure.

Reviewer 2: Discussed over Teams with @erikrobstad :
```suggestion
          To emulate play and pause, the available context type for media is added or removed. This enables and disables streaming, respectively.
```

Author: Changed as proposed.

        - The following limitations apply:

          * BAP unicast, one CIG, one of the two CIS streams (selectable).
          * Audio output: I2S (Headset).
          * Configuration: 48 kHz, 16 bit, several bit rates ranging from 24 kbps to 160 kbps.

        - Experimental

.. _software_maturity_protocol:

> 

Author: Discussed with @koffes :
Does not fit well - the other things listed are features, while the controller is a component.

> 

Author: Agreed to keep as is.
